services/api/src/routes/auth.js

```diff
@@ -106,8 +110,11 @@ router
         return;
       }
 
-      const authTokenId = generateTokenId();
-      user.authTokenId = authTokenId;
+      const token = user.addAuthInfo({
```

Comment:
just so I'm clear `addAuthInfo` always adds new token info yes?
The concept of updating (as opposed to adding) the ip/ua/etc is something we might do later?

Reply:
i think the name is bad here, perhaps user.createAuthToken({ ...info }) or something similar.

and just ignore that it does something related to a field called authInfos

Reply:
In theory the updating shouldn't really be needed, besides the IP. 

It is naturally possible the there is an update to the browser and the user agent string changes a bit, same for the mobile apps.

The auth token is somewhat device/browser specify e.g. it shouldn't travel. The goal of storing the user agent is just to make it easier for the user to identify which token is from where. And not to use for any security check.

Reply:
Well I'm not really sure if there's a point to storing the IP if we're not going to keep it updated... that said it's something that could be added later.

Reply:
ohh but we are updating it see the /users/me endpoint as you suggested i moved the update logic there